packages/material-ui/test/integration/TableCell.test.js

```diff
@@ -1,77 +1,87 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import { findOutermostIntrinsic, getClasses, createMount, createClientRender } from 'test/utils';
+import { getClasses, createClientRender, within } from 'test/utils';
 import TableCell from '@material-ui/core/TableCell';
 import TableFooter from '@material-ui/core/TableFooter';
 import TableHead from '@material-ui/core/TableHead';
 import TableBody from '@material-ui/core/TableBody';
 
 describe('<TableRow> integration', () => {
   let classes;
-  const mount = createMount();
+  const table = document.createElement('table');
   const render = createClientRender();
-  function mountInTable(node, Variant) {
-    const wrapper = mount(
-      <table>
-        <Variant>
-          <tr>{node}</tr>
-        </Variant>
-      </table>,
+  function renderInTable(node, Variant) {
```

Comment:
What about we use the `render()` API instead but with an extra <Table> component?